requirements.txt

```diff
@@ -7,7 +7,7 @@ cachetools==5.2.0
 certifi==2022.9.24
 cffi==1.15.1
 charset-normalizer==2.1.1
-cryptography==38.0.1
```

Comment:
just put <39?

Reply:
This requirements file is getting used more like a lock file.
It would make sense to use something like pip tools to separate the high level requirement from the locked requirements here.